tree/ntuple/doc/BinaryFormatSpecification.md

```diff
@@ -888,6 +888,17 @@ For example, the type name `const pair<size_t, array<class ::Event, 2>>` will be
 `std::pair<std::uint64_t,std::array<Event,2>>` with an alias type name
 `std::pair<size_t,std::array<Event,2>>`.
 
+A noteworthy implementation detail of the ROOT implemetation is the treatment of ROOT typedefs for fundamental types.
+The types `Bool_t`, `Float_t`, `Double_t`, `[U]Char_t`, `[U]Short_t`, `[U]Int_t`, `[U]Long[64]_t`
+are not treated as typedefs but they are directly mapped to the corresponding normalized RNTuple fundamental types.
+An exception is `[U]Long64_t` that appears as a template parameter of a user-defined class,
```

Comment:
Should we (again) mention `Double32_t` here? It has its own dedicated section, but maybe it's good to spell out that it is also kept in the type alias for template arguments?